reco_utils/setup.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from setuptools import setup, find_packages
+from os import path
+
+
+VERSION = __import__('__init__').VERSION
+
+here = path.abspath(path.dirname(__file__))
+
+# Get the long description from the README file
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+setup(
+    name='reco_utils',
+    version=VERSION,
+    description='Recommender System Utilities',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    url='https://github.com/microsoft/recommenders',
+    author='RecoDev Team at Microsoft',
+    author_email='RecoDevTeam@service.microsoft.com',
+    classifiers=[
+        'Development Status :: 4 - Beta',
```

Comment:
how do we decide whether to be in beta or in ` "Development Status :: 5 - Production/Stable",`??

Reply:
Not sure, there's not a lot of documentation that I can find on the distinction. To me I would say that level 5 assumes current functions will not be changed or removed (without a deprecation cycle). I suppose we could say we're there? Though I don't think we've been strictly following that up to now.